### PR TITLE
Add contributor

### LINKED ISSUE #5837: [Community] Member Profile for Senthil Athiban
**Current Behavior**

@senthil-athiban has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @senthil-athiban as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: https://github.com/senthil-athiban
- Twitter: https://x.com/senthilAthi_Ban
- LinkedIn: https://www.linkedin.com/in/senthil-athiban-m/
- Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by AlfiyaSiddique**
**Description**

This PR fixes #5837 

**Notes for Reviewers**
* Two files added to complete this task namely, `index.mdx` and profile image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/senthil-athiban/index.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+github: senthil-athiban
+twitter: senthilAthi_Ban
+linkedin: senthil-athiban-m
+layer5: f3a246ad-11f8-4a5e-85d2-8225d0c868e4
+location: Tamil Nadu, India.
+bio: Senthil Athiban is passionate about software development, with a keen interest in building products that solve real-world problems. His journey began with crafting the UI/UX of modern websites, where he developed a deep appreciation for intuitive design and user experience. He is continuously exploring the tech world, driven by an insatiable thirst for knowledge that won't be quenched until he reaches the last drop. When he's not coding or contributing to open-source projects on GitHub, you can find him on the football field, channeling his energy into his love for the game.
+badges: 
+  - layer5
+  - layer5labs
+  - meshery
```
Comment (sudhanshutech) [inline src/collections/members/senthil-athiban/index.mdx:14]: @AlfiyaSiddique add only meshery and meshmap badge

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: src/collections/members/senthil-athiban/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Senthil Athiban
+position: Contributor
+image_path: './senthil-athiban/profile.png'
```
Comment (vishalvivekm) [inline src/collections/members/senthil-athiban/index.mdx:4]:
```suggestion
image_path: './senthil-athiban/senthil-athiban.png'
```

File: src/collections/members/senthil-athiban/index.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+github: senthil-athiban
+twitter: senthilAthi_Ban
+linkedin: senthil-athiban-m
+layer5: f3a246ad-11f8-4a5e-85d2-8225d0c868e4
+location: Tamil Nadu, India.
+bio: Senthil Athiban is passionate about software development, with a keen interest in building products that solve real-world problems. His journey began with crafting the UI/UX of modern websites, where he developed a deep appreciation for intuitive design and user experience. He is continuously exploring the tech world, driven by an insatiable thirst for knowledge that won't be quenched until he reaches the last drop. When he's not coding or contributing to open-source projects on GitHub, you can find him on the football field, channeling his energy into his love for the game.
+badges: 
+  - layer5
+  - layer5labs
+  - meshery
```
Comment (vishalvivekm) [inline src/collections/members/senthil-athiban/index.mdx:14]:
```suggestion
  - meshery
```

File: src/collections/members/senthil-athiban/index.mdx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: Senthil Athiban
+position: Contributor
+image_path: './senthil-athiban/senthil-athiban.png'
+github: senthil-athiban
+twitter: senthilAthi_Ban
+linkedin: senthil-athiban-m
+layer5: f3a246ad-11f8-4a5e-85d2-8225d0c868e4
+location: Tamil Nadu, India.
+bio: Senthil Athiban is passionate about software development, with a keen interest in building products that solve real-world problems. His journey began with crafting the UI/UX of modern websites, where he developed a deep appreciation for intuitive design and user experience. He is continuously exploring the tech world, driven by an insatiable thirst for knowledge that won't be quenched until he reaches the last drop. When he's not coding or contributing to open-source projects on GitHub, you can find him on the football field, channeling his energy into his love for the game.
```
Comment (vishalvivekm) [inline src/collections/members/senthil-athiban/index.mdx:10]:
```suggestion
bio: "Senthil Athiban is passionate about software development, with a keen interest in building products that solve real-world problems. His journey began with crafting the UI/UX of modern websites, where he developed a deep appreciation for intuitive design and user experience. He is continuously exploring the tech world, driven by an insatiable thirst for knowledge that won't be quenched until he reaches the last drop. When he's not coding or contributing to open-source projects on GitHub, you can find him on the football field, channeling his energy into his love for the game."
```

**Comment by l5io** [comment]
🚀 Preview for commit bb1e61ec0b625ec3a1dd8a07552ca69f056d96b6 at: https://66d0b53dad88236125de7747--layer5.netlify.app

**Comment by AlfiyaSiddique** [comment]
The requested changes are done by @vishalvivekm. Please review again @sudhanshutech!

**Review by Ashparshp — CHANGES_REQUESTED** [review]
@AlfiyaSiddique, please compress the profile image and change it to the `.webp` format.

**Comment by Mandeep56Singh** [comment]
@AlfiyaSiddique you need to apply the changes requested by @sudhanshutech. Only then your PR can be merged.

**Comment by l5io** [comment]
🚀 Preview for commit 166c39d2509d91aaffcc3cf87d88073cceda3f72 at: https://66d33f6fa10a303d97d87dfd--layer5.netlify.app

**Comment by AlfiyaSiddique** [comment]
@Ashparshp changed the image type to `webp` and the size of image is 27 kb, let me know if it needs to be compressed more!

**Review by Ashparshp — APPROVED** [review]
LGTM! Thank you @AlfiyaSiddique.

**Review by vishalvivekm — APPROVED** [review]

**Comment by vishalvivekm** [comment]
### Yay! :balloon: :tada: :rocket:
@senthil-athiban

**Pull request merged**